cognitive_service_vision_model_customization_python_samples/models/dataset_models.py

```diff
@@ -26,21 +26,23 @@ def __init__(self, kind: AuthenticationKind = AuthenticationKind.MI, sas_token=N
 
     @property
     def params(self):
-        return {
-            'kind': self.kind.value,
-            'sasToken': self.sas_token
+        params = {
+            'kind': self.kind.value
```

Comment:
According to error coming from API, `authentication.kind` always needs to be present in the requests body